test/fuzzTesting/fuzzHelper.cpp

```diff
@@ -143,16 +151,136 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
 	{
 		switch (_opcode)
 		{
+		case dev::eth::Instruction::PUSH1:	code += rndByteSequence(1);	break;
```

Comment:
I guess it would be better to use something like
```
if (int(dev::eth::Instruction::PUSH1) <= int(_opcode) && int(_opcode) <= int(dev::eth::Instruction::PUSH32))
  code += rndByteSequence(int(_opcode) - int(dev::eth::Instruction::PUSH1) + 1);
```